Tutorial_1b_Moment_calculation.ipynb

```diff
@@ -0,0 +1,860 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    rms_spec_realnoise = mad_std(spec_noise_realistic[spec_noise_realistic&lt;3*rms_spec])
</pre><p>Worth a quick note that, should you have a large number of spectra (i.e. a cube), you could calculate the std per channel or group of channels to come up with a noise spectrum.</p><p> </p><p>Also consider using the astropy sigma_clip routine to minimize the signal contribution: <a href="https://docs.astropy.org/en/stable/api/astropy.stats.sigma_clip.html" target="_blank">https://docs.astropy.org/en/stable/api/astropy.stats.sigma_clip.html</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>